src/components/ScholarshipForm.jsx

```diff
@@ -229,6 +230,10 @@ function ScholarshipForm({ scholarship, submitFn, onSubmitError }) {
     </Grid>
   );
 
+  stepperItems.Review = (
```

Comment:
You should prevent the **Next** button from being clickable if there are form errors. I just tried clicking to the end and it crashed trying to render the detail view.

Reply:
I was gonna add this in another PR but I added it here.